action.yml

```diff
@@ -196,11 +196,13 @@ runs:
           exit 1
         fi
       env:
+        GEMINI_MODEL: '${{ inputs.gemini_model }}'
```

Comment:
Please define these inputs higher in the file. You cannot reference undefined inputs.

Reply:
Thanks!